addons/website_forum/controllers/main.py

```diff
@@ -153,10 +167,12 @@ def tags(self, forum, tag_char=None, **post):
         first_char_tag = forum.get_tags_first_char()
         first_char_list = [(t, t.lower()) for t in first_char_tag if t.isalnum()]
         first_char_list.insert(0, (_('All'), 'all'))
-        # get active first char tag
-        active_char_tag = first_char_list[1][1] if len(first_char_list) > 1 else 'all'
+
         if tag_char:
             active_char_tag = tag_char.lower()
+        else:
+            active_char_tag = 'all'
```

Comment:
Same tha for blog
`active_char_tag = tag_char and tag_char.lower() or 'all'
`
Sorry, I know that you don't like comment on python code (what Jonathan say Friday)

Reply:
Actually I'm learning a lot, so thanks so much for your time and your patience :)

I'm trying to be helpful because you have a lot to review ( I wrote that in the pad, in the blog's PR etc...), my python code is there ONLY to get the visual result and to allow me to layout with real data.

**Just scrap everything if you prefer, I don't mind as long long as it keep the same visual result :)**

If you have the feeling that you're "wasting too much time" with me, just share your thoughts ;)

Reply:
> If you have the feeling that you're "wasting too much time" with me, just share your thoughts ;)

Really not, I think that we help you to improve your python knowledge for future branch...
While it seems that you take it like if we want bother you. What is really not the case.

As you said, if you prefer, I can refactor myself your python code if you prefer. But it will take, of course, more time for us since I don't know always the impact of your python code or what you was trying todo and because instead to explain to you one time, I need to fix each branch... All is good for me...